packages/ilios-common/addon/components/editable-field.hbs

```diff
@@ -70,33 +77,41 @@
                   />
                 {{/if}}
               </button>
-              {{#if isTruncated}}
-                <EllipsisIcon />
-                <button
-                  class="expand-text-button"
-                  type="button"
-                  aria-label={{t "general.expand"}}
-                  title={{t "general.expand"}}
-                  data-test-expand
-                  {{on "click" expand}}
+              {{#if isFaded}}
+                <div
+                  class="fade-text-control"
+                  data-test-fade-text-control
+                  {{! template-lint-disable no-invalid-interactive}}
+                  {{on "click" (fn this.setIsEditing true)}}
```

Comment:
This is the part I get worried about the fairly large API needed to call this in the block form. This is a button now in the component, but has remanned a clickable div here. I'm thinking passing the button itself in as a parameter might work here? But I can't recall if I've ever done that anywhere.

Reply:
I got this template from `<TruncateText>`, which does a similar thing, so figured it was kosher for now. Is this change required for merging?

Reply:
I hate to move on, but this thing is already pretty big. I accept this, let's keep it for now.